charts/blockstream-simulator/templates/deployment.yaml

```
@@ -32,6 +32,7 @@ spec:
         - name: data-storage
           persistentVolumeClaim:
             claimName: {{ if .Values.simulator.persistence.data.create }}{{ include "blockstream-simulator-chart.fullname" . }}-data{{ else }}{{ .Values.simulator.persistence.data.existingClaim }}{{ end }}
+      {{- if eq .Values.simulator.config.BLOCK_STREAM_SIMULATOR_MODE "PUBLISER_MODE" }}
```

Should this be publis**h**er?
```suggestion
      {{- if eq .Values.simulator.config.BLOCK_STREAM_SIMULATOR_MODE "PUBLISHER_MODE" }}
```